modules/core/src/main/java/org/apache/ignite/internal/processors/platform/utils/PlatformConfigurationUtils.java

```diff
@@ -622,6 +622,8 @@ public static void readIgniteConfiguration(BinaryRawReaderEx in, IgniteConfigura
         ClientListenerProtocolVersion ver) {
         if (in.readBoolean())
             cfg.setClientMode(in.readBoolean());
+        if (in.readBoolean())
```

Comment:
As far as I remember, *nullable* properties should be protected with `in.readBoolean()` construct. We need to double check it with .NET contributors. Also, the added property should be added to a .NET configuration mirror `modules/platforms/dotnet/Apache.Ignite.Core/IgniteConfiguration.cs`